packages/client-graphql/src/initial-state.ts

```diff
@@ -0,0 +1,97 @@
+import { Client } from "urql";
+import { brandProps } from "./brands";
+import { createCourierClient } from "./client";
+import { ICourierClientBasicParams } from "./types";
+
+interface IInitialStateParams {
+  skipFetchBrand?: boolean;
+  brandId?: string;
+  from?: number;
+}
+
+type GetInitialState = (params: IInitialStateParams) => Promise<
+  | {
+      brand: any;
+      unreadMessageCount: number;
+    }
+  | undefined
+>;
+
+const buildQuery = ({
+  skipFetchBrand,
+  brandId,
+}: Partial<IInitialStateParams>) => {
+  if (skipFetchBrand) {
+    return `
+      query InitialState($params: FilterParamsInput) {
```

Comment:
Is it possible to use `directives` here in place of string interpolation here? Just thinking aloud.

Reply:
Apparently not :/

Reply:
Only supports fields.